src/classes/HH_INaming.cls

```diff
@@ -69,11 +69,11 @@ global Interface HH_INaming {
     * @param listCon the list of Contacts in the Household
     * @return the appropriate Household name or greeting
     */ 
-    string getExampleName(Household_Naming_Settings__c hns, string strField, list<Contact> listCon);
+    String getExampleName(Household_Naming_Settings__c hns, String strField, List<Contact> listCon);
     
     /*******************************************************************************************************
     * @description To allow the NPSP code to know which fields to query for on Contact, before handing them off to the naming routines.
     * @return set of strings of all fields referenced from Contact.
     */
-    set<string> setHouseholdNameFieldsOnContact();
+    Set<String> householdNamingFields();
```

Comment:
I just realized this is in a global class. We probably can't change the signature on this method as a result since customers may have implemented it. The downsides of global in a managed package.

Reply:
Sounds good, just changed it back.